README.md

```diff
@@ -1,6 +1,13 @@
 # tree-hugger
 Mining source code repositories at scale easily. Tree-hugger is a light-weight, high level library which provides Pythonic APIs  to mine recursively trough Github repositories.
-Tree-hugger is built on top of tree-sitter and covers Python and PHP source code. Coming soon: Java ana d JavaScript.
+Tree-hugger is built on top of tree-sitter and covers Python and PHP source code. 
```

Comment:
As we cover more than that, maybe remove the names of the languages from this line.